Index: src/main/java/pl/bel/comparable/StartApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package pl.bel.comparable;\r\n\r\nimport java.util.TreeSet;\r\n\r\npublic class StartApplication {\r\n    public static void main(String[] args) {\r\n        TreeSet<Motor> myMotorList = new TreeSet<>();\r\n        Motor suzuki = new Motor(3, 3500, \"Blue\", true);\r\n        Motor honda = new Motor(7, 3200, \"Red\", true);\r\n        Motor harley = new Motor(2, 5400, \"Gold - Yellow\", false);\r\n        myMotorList.add(suzuki);\r\n        myMotorList.add(honda);\r\n        myMotorList.add(harley);\r\n\r\n        for (Motor m : myMotorList) {\r\n            System.out.println(m);\r\n        }\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/pl/bel/comparable/StartApplication.java b/src/main/java/pl/bel/comparable/StartApplication.java
--- a/src/main/java/pl/bel/comparable/StartApplication.java	(revision 9fcb3b518c7334c17d562a1594637038907d80eb)
+++ b/src/main/java/pl/bel/comparable/StartApplication.java	(date 1650053387323)
@@ -1,5 +1,7 @@
 package pl.bel.comparable;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.TreeSet;
 
 public class StartApplication {
@@ -12,8 +14,10 @@
         myMotorList.add(honda);
         myMotorList.add(harley);
 
+        List<Motor> listMotors = new ArrayList<>();
+
         for (Motor m : myMotorList) {
             System.out.println(m);
         }
     }
-}
+}
\ No newline at end of file
